webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -67,6 +70,10 @@ function DraftRow({
     isRemote,
     scrollIntoView,
 }: Props) {
+    const [isEditing, setIsEditing] = useState(false);
+
+    const teamId = useSelector(getCurrentTeamId);
```

Comment:
I may be missing something, but I don't see the `teamId` used anywhere else. Why did you move it out of the other selector?

Reply:
Right, there's no need now. It was used intermediately in some refactoring but not now. Removed.